apiml/src/main/resources/apiml-log-messages.yml

```diff
@@ -0,0 +1,24 @@
+messages:
+    # Info messages
+    # 000-099
+
+    # General messages
+    # 100-199
+
+    # HTTP,Protocol messages
+    # 400-499
+
+    # TLS,Certificate messages
+    # 500-599
+
+    # Various messages
+    # 600-699
+
+    # Service specific messages
+    # 700-999
+    -   key: org.zowe.apiml.common.unauthorized
```

Comment:
maybe this import is not necessary, this message is already in `common-log-messages.yml` which is already included in the `MessageService` bean, but we should verify it